src/index.js

```diff
@@ -67,15 +67,22 @@ export default function VeeValidatePlugin (opts) {
         }
       }
 
-      return {
-        ...el,
-        // namespaced prop to avoid clash with users' props
-        _veeValidateConfig: {
-          mapProps,
-          path
-        },
-        component: withField(el)
+      const constructField = (f) => {
+        return {
+          ...f,
```

Comment:
Would you mind switching these `f` to `field`? For clarity :)

Reply:
Sure. a unit test is added for it. let me know if there are more cases needed to be covered.